src/System.Reflection.Metadata.ILDasm/src/Decoder/ILDecoder.cs

```diff
@@ -0,0 +1,582 @@
+﻿using ILDasmLibrary.Instructions;
+using Microsoft.CSharp;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Decoding;
+using System.Reflection.Metadata.Ecma335;
+using System.Text;
+
+namespace ILDasmLibrary.Decoder
```

Comment:
Name will be System.Reflection.Metadata.CIL
Decided today in a Design Review that will be uploaded later.

And I'm updating all Namespaces to match that name.